tests/System.Buffers.Primitives.Tests/ReadableBufferFacts.cs

```diff
@@ -140,7 +140,7 @@ public void SegmentStartIsConsideredInBoundsCheck()
         }
 
         [Fact]
-        public void MovePrefersNextSegment()
```

Comment:
Nit: might as well rename other test methods containing "Move" in the name.

Reply:
Fixed